locallib.php

```diff
@@ -125,6 +125,79 @@ function hvp_get_core_assets() {
 }
 
 /**
+ * Add required assets for displaying the editor.
+ *
+ * @param int $id Content being edited. null for creating new content
```

Comment:
Id from the `mdl_hvp` table, right? Might be useful to clarify it. I remember being confused about that the first time I was reading the plugin code.

Moodle has module id, course module id, activity id, etc. It's easy to get confused. :)